packages/navie/src/services/message-token-reducer-service/token-count-strategies/estimate-token-count.ts

```diff
@@ -0,0 +1,10 @@
+import Message from '../../../message';
+import { TokenCountStrategy } from '../interfaces';
+
+export default class EstimateTokenCount implements TokenCountStrategy {
+  constructor(private readonly charactersPerToken = 3) {}
```

Comment:
We could use the constant we already have defined somewhere so it doesn't grow inconsistent.